spring-cloud-common-security-config/spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/OAuthSecurityConfiguration.java

```diff
@@ -216,28 +216,23 @@ protected void configure(HttpSecurity http) throws Exception {
 			http.addFilter(basicAuthenticationFilter);
 		}
 
-		this.authorizationProperties.getAuthenticatedPaths().add("/");
-		this.authorizationProperties.getAuthenticatedPaths()
-				.add(dashboard(authorizationProperties, "/**"));
-		this.authorizationProperties.getAuthenticatedPaths()
-				.add(this.authorizationProperties.getDashboardUrl());
-		this.authorizationProperties.getPermitAllPaths()
-				.add(this.authorizationProperties.getDashboardUrl());
-		this.authorizationProperties.getPermitAllPaths()
-				.add(dashboard(authorizationProperties, "/**"));
-		ExpressionUrlAuthorizationConfigurer<HttpSecurity>.ExpressionInterceptUrlRegistry security =
+		//this.authorizationProperties.getAuthenticatedPaths().add("/");
+		//this.authorizationProperties.getAuthenticatedPaths().add(this.authorizationProperties.getDashboardUrl());
+		//this.authorizationProperties.getAuthenticatedPaths().add(dashboard(authorizationProperties, "/**"));
```

Comment:
When I uncomment those lines I can't reach the Angular login page even If I add it some lines below. as `permitAllPaths` via `this.authorizationProperties.getPermitAllPaths().add(authorizationProperties.getLoginUrl());`

Reply:
Maybe resolved by: https://github.com/klopfdreh/spring-cloud-dataflow/pull/2

Reply:
I think this PR should be correct now. I changed the other to `authenticated` > `permitAll` > `anonymous`.

So for all pages the user should be authenticated and after that it is permitted to all users no matter their role and at last the login page is allowed for anonymous so that we can access it without being logged in.

I resolve this conversation as I think that the previously mentioned PR is correct, now.